websites/W/Wbijam/presence.ts

```diff
@@ -0,0 +1,151 @@
+const presence = new Presence({
+		clientId: "992829484564746290",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameVideo: boolean,
+	currentTime: number,
+	duration: number,
+	paused: boolean;
+
+interface IFrameData {
+	iframeVideo: {
+		dur: number;
+		iFrameVideo: boolean;
+		paused: boolean;
+		currTime: number;
+	};
+}
+
+presence.on("iFrameData", (data: IFrameData) => {
+	if (data.iframeVideo.dur) {
+		({ iFrameVideo, paused } = data.iframeVideo);
+		currentTime = data.iframeVideo.currTime;
+		duration = data.iframeVideo.dur;
```

Comment:
Consistency
```suggestion
		{ iFrameVideo, paused, currentTime, dur } = data.iframeVideo;
```

Reply:
imo previous code is better option
![image](https://user-images.githubusercontent.com/63014799/180398924-bf288402-fb30-45b0-a36c-fceac6586674.png)

Reply:
![image](https://user-images.githubusercontent.com/63014799/180399667-e1cefe1d-883c-4e63-914a-dea6d378308f.png)